Revision: 34700

ConnellyJHW opened the pull request:
【BUG】【new】【OM支持SequoiaSQL-OLTP】创建SequoiaSQL-OLTP业务页面中的参数信息有错误
【重现步骤】 
1、使用OM创建SequoiaSQL-OLTP业务，发现业务配置参数的提示信息存在错误或者参数的提示信息、格式等不正确 
【预期结果】 
参数的提示信息正确，提示信息中说明参数含义，对于使用默认值的参数，在提示信息中指出默认值及其意义 
【实际结果】 
业务配置参数信息不完全 
【备注】 
存在问题的参数参考附件config.docx

EmilianLLJH commented:
请安排分析

FlorentinoJH commented:
请处理

ConnellyJHW commented:
【修改描述】 
已根据官网的配置项说明修改描述不清楚的配置项的参数提示信息。 

【提交记录】 
Message: 
3014 
1.调整部署SequoiaSQL-OLTP业务页面中的配置项参数提示信息。 
```
Modified : /trunk/SequoiaDB/web/config/sequoiasql-oltp_config_en.xml 
Modified : /trunk/SequoiaDB/web/config/sequoiasql-oltp_config_zh-CN.xml 
```

【CheckList】 
检查项---------------------------------------------------------结论（NA/N/Y） 
修改代码是否全部提交SVN主干和分支版本 
且符合代码规范--------------------------------------------y 主干 
是否用例全部通过-----------------------------------------n sac网页手工测试 
修改代码是否通过检视------------------------------------n 
资料是否变更-----------------------------------------------n 
接口是否变更-----------------------------------------------n 
相关驱动是否变更或添加---------------------------------n 
是否保持向前兼容------------------------------------------y 
是否影响安装部署和监控---------------------------------n 
是否影响性能-----------------------------------------------n 
sdb shell帮助信息是否更新------------------------------n

FlorentinoJH commented:
请安排测试

EmilianLLJH commented:
请验证

BrigidLLX commented:
【验证结果】验证不通过 
【备注】 
client encoding、log destination、log error verbosity、work mem参数的描述信息仍存在错误

EmilianLLJH commented:
还有问题

FlorentinoJH commented:
请处理

ConnellyJHW commented:
补充修改client encoding、log destination、log error verbosity、work mem参数的描述信息。 

【提交记录】 
Message: 
3014 
1.修正部署OLTP业务时配置项的提示信息错误：client_encoding、log_error_verbosity、log_destination、work_men 
```
Modified : /trunk/SequoiaDB/web/config/sequoiasql-oltp_config_en.xml 
Modified : /trunk/SequoiaDB/web/config/sequoiasql-oltp_config_zh-CN.xml 

```
【CheckList】 
检查项---------------------------------------------------------结论（NA/N/Y） 
修改代码是否全部提交SVN主干和分支版本 
且符合代码规范--------------------------------------------y 主干 
是否用例全部通过-----------------------------------------n sac网页手工测试 
修改代码是否通过检视------------------------------------n 
资料是否变更-----------------------------------------------n 
接口是否变更-----------------------------------------------n 
相关驱动是否变更或添加---------------------------------n 
是否保持向前兼容------------------------------------------y 
是否影响安装部署和监控---------------------------------n 
是否影响性能-----------------------------------------------n 
sdb shell帮助信息是否更新------------------------------n

FlorentinoJH commented:
请安排测试

EmilianLLJH commented:
请验证

DaelenLL commented:
【验证结果】： 回归不通过 
【回归步骤】 
log_error_verbosity默认配置项不正确

EmilianLLJH commented:
验证不通过

FlorentinoJH commented:
请确认回归结果

ConnellyJHW commented:
log_error_verbosity配置项中的下拉菜单选项错误，正确的选项应该是TERSE、DEFAULT和VERBOSE。 

【修改记录】 
Message: 
3014 sac 
1.修复创建PostgreSQL业务时，log_error_verbosity参数选项错误的问题。 
```
Modified : /trunk/SequoiaDB/web/config/sequoiasql-postgresql_config_en.xml 
Modified : /trunk/SequoiaDB/web/config/sequoiasql-postgresql_config_zh-CN.xml 
```

【CheckList】 
检查项---------------------------------------------------------结论（NA/N/Y） 
修改代码是否全部提交SVN主干和分支版本 
且符合代码规范--------------------------------------------Y 主干 
是否用例全部通过-----------------------------------------N sac手工测试 
修改代码是否通过检视------------------------------------NA 
资料是否变更-----------------------------------------------N 
接口是否变更-----------------------------------------------N 
相关驱动是否变更或添加---------------------------------N 
是否保持向前兼容------------------------------------------Y 
是否影响安装部署和监控---------------------------------N 
是否影响性能-----------------------------------------------N 
sdb shell帮助信息是否更新------------------------------NA

FlorentinoJH commented:
请安排测试

Baberir commented:
请验证

CristianZZY commented:
【验证结果】：验证3.0.1版本验证问题已经解决 ，请检视  
【验证步骤】 
1.进入“部署”页面 
2.点击“添加业务”按钮，选择“创建业务” 
3.在“创建业务”页面中“业务类型”选择“SequoiaSQL-PostgreSQL” 
4.在“配置业务”页面中选择“error diagnosis”标签选项 
5.查看参数“log error verbosity”下拉菜单显示三种选项分别为“TERSE”、“DEFAULT”和“VERBOSE”，参数信息正确

Baberir commented:
检视ok，该问题可关闭
